pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -162,6 +171,30 @@ func (r *Runtime) GlobalParamDescs() params.ParamDescs {
 			DefaultValue: fmt.Sprintf("%d", ConnectTimeout),
 			TypeHint:     params.TypeUint16,
 		},
+		{
+			Key:          ParamTLSKey,
+			Description:  "TLS client key",
+			DefaultValue: "",
+			TypeHint:     params.TypeString,
+		},
+		{
+			Key:          ParamTLSCert,
+			Description:  "TLS client certificate",
+			DefaultValue: "",
+			TypeHint:     params.TypeString,
+		},
+		{
+			Key:          ParamTLSServerCA,
+			Description:  "TLS server CA certificate",
+			DefaultValue: "",
+			TypeHint:     params.TypeString,
+		},
+		{
+			Key:          ParamTLSServerName,
+			Description:  "override TLS server name (if omitted, using target server name)",
+			DefaultValue: "",
+			TypeHint:     params.TypeString,
+		},
```

Comment:
Shouldn't they be moved under `case ConnectionModeDirect` to avoid showing them in the kubernetes case?

Reply:
At least until we also support it for ig-k8s, yes.